app/assets/javascripts/listings/components/listing/features-section.html.slim

```diff
@@ -72,26 +71,22 @@ accordion-heading
 
   .content-tile.feature-tile ng-if="$ctrl.parent.isRental($ctrl.parent.listing)"
     .content-card.bg-dust
-      h5.content-card_title Additional Fees
+      h5.content-card_title translate="listings.features.additional_fees"
       .row.collapse.margin-bottom
         .small-6.columns.padding-right.padding-bottom
-          span.t-small Application Fee
+          span.t-small translate="listings.features.additional_fee"
           p.t-delta.no-margin.t-bold
             | {{::$ctrl.parent.listing.Fee | currency:"$":2}}
-          span.content-label.t-small
-            | per applicant age 18 and over
-          span.t-tiny.d-block
-            | Due post lottery
+          span.content-label.t-small translate="listings.features.per_applicant_age_18_and_over"
+          span.t-tiny.d-block translate="listings.features.due_post_lottery"
         .small-6.columns.padding-bottom
-          span.t-small Deposit
+          span.t-small translate="listings.features.deposit"
           p.t-delta.no-margin.t-bold
             | {{::$ctrl.parent.listing.Deposit_Min | currency:"$":0}}
             span ng-if="$ctrl.parent.listing.Deposit_Max && $ctrl.parent.listing.Deposit_Max != $ctrl.parent.listing.Deposit_Min"
               | &ndash;{{ $ctrl.parent.listing.Deposit_Max | currency:"$":0 }}
-          span.content-label.t-small
-            | or one month's rent
-          span.t-tiny.d-block ng-if="$ctrl.listingIsBMR()"
-            | May be higher for lower credit scores
+          span.content-label.t-small translate="listings.features.or_one_months_rent"
```

Comment:
this bit might be tough too

Reply:
Do you want the hole thing including deposit in a translation?

Reply:
This is probably fine as is. We'll see if we hear any issues from the translators